### DISCUSSION TITLE
Week 2 Individual assignment

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#18] [Assignments]
## In-class Exercise (also your individual assignment)

Now it is your turn to put all this newfound javascript knowledge into practice.

1. In your `index.html` file, and in the javascript section of your code (`<script></script>`), create an array of objects that lists locations that you have travelled to (include at least 5 locations). Make sure to include a title, description, latitude, and longitude for each.
1. Replace the code that generates the single marker with a loop that goes through your array of objects. Then, create a marker for each location. Make sure to include a title and description in the popup.
1. Add a title of your maproom in the header section.
1. Add a title and description of your map in the sidebar section.
1. When you are done, upload your additions to your GitHub Repo ([follow these instructions to commit and push from the week 1 lab](https://github.com/yohman/21S-DH151/tree/main/Weeks/Week01/Lab#push-your-new-files-to-your-github-repo))
1. Add the GitHub Pages URL to your maproom in the class discussion section

**Comment by kimjee8955** [comment]
https://kimjee8955.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
So global! Where in Osaka did you live?

**Comment by michelle14lee** [comment]
https://michelle14lee.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
$1 pizza!

**Comment by lisalou36** [comment]
https://lisalou36.github.io/DH151//week%202/index.html

**Reply by yohman** [reply]
What a road trip! Might be good to draw lines between them, and add photos that you took.

**Comment by ivanma9** [comment]
https://ivanma9.github.io/DH151/Week2/

**Reply by yohman** [reply]
Nice images! And dynamic sidebar, well done :+1:

**Comment by eliaranita1** [comment]
https://eliaranita1.github.io/DH151/week%202/index.html

**Reply by yohman** [reply]
Love the clickable photo markers ! :+1:

**Comment by justisg** [comment]
https://justisg.github.io/DH151/Week2/

**Reply by yohman** [reply]
I had tickets to go to the Tokyo Olympics, but sadly had to get them refunded. You do have the Tokyo marker coded twice :)

**Comment by alicelu170** [comment]
https://alicelu170.github.io/DH151/Week%202/index.html

**Reply by yohman** [reply]
What a global journey! And nice touch in making the sidebar larger :+1:

**Comment by lisa58** [comment]
https://lisa58.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
Nice images! I like how you created your popups.  Not sure what your `heading` and `desc` variables do?

**Comment by derekwang99** [comment]
https://derekwang99.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
Nice map that shows your life journey.

**Comment by hanarama** [comment]
https://hanarama.github.io/DH151/Week%202/index.html

**Reply by yohman** [reply]
Congratulations! Love the honeymoon map. Would be great to see the after honeymoon map with pictures!

**Comment by JuliaWood1** [comment]
https://juliawood1.github.io/DH151/Week2/index.html

**Comment by alexthegreat1** [comment]
https://alexthegreat1.github.io/DH151/Week%202/index.html

**Reply by yohman** [reply]
Nice adjustment to your css (easy to read). Also, love how you coded your popup `<h1>${city.title}</h1><p>${city.description}</p>`  👍

**Comment by danielliuzy** [comment]
https://danielliu524.github.io/DH151/Week2/index.html

**Comment by adrianaromero819** [comment]
https://adrianaromero819.github.io/DH151/Week2/index.html

**Comment by zonaliao** [comment]
https://zonaliao.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
Nice global map! But let's put the markers _within_ the loop so as not to replicate a lot of code.

**Comment by alekhyavittalam** [comment]
https://alekhyavittalam.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
Nice images! Would love to see pictures _you_ took at those locations.

**Comment by jomh17** [comment]
https://jomh17.github.io/DH151/WEEK%202/index.html

**Comment by frida-barragan** [comment]
https://frida-barragan.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
:open_mouth: Wow! 🥇

**Comment by natgrace** [comment]
https://natgrace.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
Amazing photo journal. More, more!

**Comment by anncong** [comment]
https://anncong.github.io/DH151/Week2/Index.html

**Comment by kaisanwatanabe** [comment]
https://kaisanwatanabe.github.io/DH151/week%202/index.html

**Reply by yohman** [reply]
Nice pictures. Born in Rome!

**Comment by kmcinerny** [comment]
https://kmcinerny.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
Beautiful photos! And nice zoomIn function 👍

**Comment by LynnT2** [comment]
https://LynnT2.github.io/DH151/Week%202/index.html

**Reply by yohman** [reply]
Yes, UCLA! Nice fonts :)

**Comment by ryanorlando** [comment]
https://ryanorlando.github.io/Dh151/Week2/index.html

**Reply by yohman** [reply]
🏀 !!

**Comment by ainsleighdouglas** [comment]
https://ainsleighdouglas.github.io/DH-151/Week2/index.html

**Reply by yohman** [reply]
Wow 🏊‍♀️

**Comment by hoz-map** [comment]
https://hoz-map.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
Nice images, and nice adjustment to your css

**Comment by kaicolorado** [comment]
https://kaicolorado.github.io/DH151/Week%202/index.html

**Comment by yjwsb233** [comment]
https://yjwsb233.github.io/DH151/Week2/index.html

**Comment by jmtanenbaum** [comment]
https://jmtanenbaum.github.io/DH151/week2/index.html

**Comment by ec028** [comment]
https://ec028.github.io/DH151/Week2/index.html

**Reply by yohman** [reply]
Nice. No need to add the initial marker (Trinity College) twice.

**Comment by MaryJaneHowland** [comment]
https://maryjanehowland.github.io/DH151/Week%202/index.html
